Automattic/wp-calypso
# i18n: Localize links in Jetpack app promo sections

Related to:
* 555-gh-Automattic/i18n-issues
* D101286-code

## Proposed Changes

* Localize link and QR code for Jetpack app in Full Width App Promo component.
* Localize link for Jetpack app in my sites customer home card.

![CleanShot 2023-02-14 at 14 28 47](https://user-images.githubusercontent.com/2722412/218740097-832cd2f6-547a-4add-8b08-d93b00dfb6a1.png)

![CleanShot 2023-02-14 at 14 34 58](https://user-images.githubusercontent.com/2722412/218740359-2b902ff1-0c0e-4443-9384-a66cddc4e32b.png)


## Testing Instructions


* Boot Calypso locally or use calypso.live image.
* Change UI to any of the Mag-16 languages.
* Navigate to `/home/`
* Confirm `Get the Jetpack app` links to `apps.wordpress.com/[locale]/get/`.
* Navigate to `/stats/day/[site]`
* Confirm `wp.com/app` and QR code link to `apps.wordpress.com/[locale]/get/`.

## Pre-merge Checklist
